requirements.txt

```diff
@@ -0,0 +1,5 @@
+py==1.4.34
```

Comment:
I see the confusion, there was already a `dev-requirements.txt`. You can remove `requirements.txt` and then the change you made to `dev-requirements.txt` is perfect.